acceptance-tests/proxy_protocol_test.go

```diff
@@ -61,9 +60,51 @@ var _ = Describe("Proxy Protocol", func() {
 		By("Sending a request without Proxy Protocol Header to HAProxy health check port")
 		_, err = http.Get(fmt.Sprintf("http://%s:8080/health", haproxyInfo.PublicIP))
 		expectConnectionResetErr(err)
+
 	})
 })
 
+var _ = Describe("Proxy Protocol with specified expect_proxy and accept_proxy=false", func() {
+	opsfileProxyProtocol := `---
+# Enable Proxy Protocol for specified CIDRs
+- type: replace
+  path: /instance_groups/name=haproxy/jobs/name=haproxy/properties/ha_proxy/accept_proxy?
+  value: false
+- type: replace
+  path: /instance_groups/name=haproxy/jobs/name=haproxy/properties/ha_proxy/expect_proxy?
+  value:
+    - 127.0.0.1/8
+    - ::1/128
```

Comment:
@peanball Of course, we are not ready with the tests at all. We have only one without any meaning at first and still trying to execute it. The tests are not ready for review at all. We just pushed this stand because it was required by the tests.